build.sbt

```diff
@@ -81,6 +83,8 @@ lazy val `kyo-scheduler` =
         .in(file("kyo-scheduler"))
         .settings(
             `kyo-settings`,
+            scalacOptions -= "-language:strictEquality",
```

Comment:
It should be possible to configure scalacOptions per compiler version.

Maybe it's worth adding `-Xsource:3` for Scala 2.13.

https://github.com/scala/scala/releases/tag/v2.13.13

Reply:
I've tried a few ways but sbt is always a big mystery for me. I've added the flag in a naive way and it's working but sbt is emitting a warning. I think it's ok for now.